assets/css/custom.css

```diff
@@ -114,23 +124,62 @@ body {
   position: relative;
 }
 
+.letters {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 30%;
+}
+
+.letters img {
+  height: 100%;
+}
+
+@media only screen and (min-width: 769px) {
+  .bag-custom {
+    width: calc(40vh * 1.46);
+    height: 40vh;
+  }
+  .letters {
+    width: 100%;
+    text-align: center;
+  }
+}
+
+input[type="text"].letters-input {
+  border: 1px solid black;
+  display: inline-block;
+  text-align: center;
+  text-transform: uppercase;
+  text-indent: 0;
+  width: 2.5em;
+  padding: .5em 0;
+}
+
+
+/* Tabs */
+
 .tab-cnr {
   display: flex;
+  z-index: 1200;
```

Comment:
no need to use such a high z-index. `z-index: 1` should do it since we're not using z-index anywhere else.